packages/devtools_app/test/inspector_v2/inspector_integration_test.dart

```diff
@@ -508,3 +542,11 @@ Finder findExpandCollapseButtonForNode({
 
   return expandCollapseButtonFinder;
 }
+
+Finder findSearchButton() {
```

Comment:
nit: make this private, or just inline the `find.ancestor` finder above where this method is used

Reply:
done! moved it inline